src/serialchemy/nested_fields.py

```diff
@@ -162,8 +161,19 @@ def get_model_pk_attr_name(model_class):
 
     :return: str: a Column name
     """
-    from sqlalchemy.inspection import inspect
-    primary_key_names = [pk for pk in inspect(model_class)._primary_key_propkeys]
+    import sys
+
+    # TODO EDEN-2586: Investigate SqlAlchemy inspect failing on Python 3.6
+    if sys.version.startswith('3.6'):
```

Comment:
Of course the previous code works, but this is more robust in general, specially for other version checks.

```suggestion
    if sys.version_info[:2] == (3, 6):
```